index.html

```diff
@@ -193,20 +194,27 @@ <h2 class="conta">CONTACT US</h2>
     if you have an application you are interested in developing, a feature that you need built or a project that needs coding. I'd love to help with it
 </p>
 
-<form >
-    <input class="text-inp" type="text" placeholder=" Text Input">
-
-    <input class="email" type="text" placeholder=" Jeremy@gmail.com">
-
-    <textarea class="send-text" cols="30" rows="10" placeholder="
-Enter text here"></textarea>
-</form>
-
-<button type="button" class="btn-sec"> See Project </button>
-
-<div class="cont-line">.</div>
-
-
+    <form action="https://formspree.io/f/mlezygyr" method="post">
+        <ul class="input-list">
+         <li>
+           <label for="name">Name:</label>
+           <input type="text" id="name" name="user_name" placeholder="  Name" maxlength="30" required>
+         </li>
+         <li>
+           <label for="mail">E-mail:</label>
+           <input type="email" id="mail" name="user_email" placeholder="  E-mail" required>
+         </li>
+         <li>
+           <label for="msg">Message:</label>
+           <textarea cols="33" rows="16"  id="msg" name="user_message" 
+           placeholder=" 
+ Enter text here" maxlength="500" required ></textarea>
+         </li>
+         <li class="button">
+           <button type="submit" id="form-btn"> see project </button>
+         </li>
+        </ul>
+       </form>
```

Comment:
- [ ] As suggested by the previous reviewer, please kindly follow the Figma design templates as there are no labels in the design template.

| Your Design  | Template Design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/52934128/177458915-1dce5534-9f24-47c6-893f-0f607602abba.png)  |![image](https://user-images.githubusercontent.com/52934128/177458954-dc5f8f07-a78c-439b-b5fa-11d3e16b574b.png)  |